lunatrace/bsl/hasura/migrations/lunatrace/1664218225933_release_analysis_result/up.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TYPE finding_state AS ENUM (
+    'vulnerable',
+    'not_vulnerable'
+    );
+
+CREATE TABLE package.release_analysis_result
+(
+    id                          uuid PRIMARY KEY DEFAULT (gen_random_uuid()),
+    manifest_dependency_edge_id uuid,
+    vulnerability_id            uuid,
+    state                       finding_state,
```

Comment:
shouldn't basically ALL of these be NOT NULL?

Reply:
:+1: